### DISCUSSION TITLE
Upgrade guide v1 to v2?

### DISCUSSION THREAD

**Discussion by tavin** [awspring/spring-cloud-aws#328] [General]
Is there an upgrade guide somewhere, or a list of breaking changes?

**Comment by maciejwalkowiak** [comment]
Since 3.0 (based on v2) is still in snapshots, there is no upgrade guide. It will be prepared for each milestone release. 

FYI 3.0 will be a massive breaking change and getting familiar with the reference would be a must. You can find current state of docs here: https://docs.awspring.io/spring-cloud-aws/docs/3.0.0-SNAPSHOT/reference/html/index.html and track progress with 3.0 M1: https://github.com/awspring/spring-cloud-aws/milestone/4

**Reply by tavin** [reply]
Ok thank you. Right now I'm trying to help my company move from v1 to v2 though, and I couldn't find any migration guide.

**Reply by maciejwalkowiak** [reply]
Do you mean Spring Cloud AWS 1.x to 2.x? It's been before I've joined this project. Original maintainers are not there anymore.

**Reply by tavin** [reply]
Yes that's what I mean. Just hoping somebody might know :)
